packages/lights.yaml

```diff
@@ -222,22 +223,42 @@ automation:
             }}
 
     action:
-      - delay:
-          seconds: |
-            {{ trigger.event.data.new_state.attributes.off_delay_seconds | default(1) | int }}
-      # Double check that the sensor is still off after the delay to avoid races
+      - service: system_log.write
+        data_template:
+          message: >
+            at=delay_start
+            entity_id={{ trigger.event.data.entity_id }}
+            delay={{ trigger.event.data.new_state.attributes.off_delay_seconds | default(0) | int }}
+          logger: packages.lights
+          level: info
+      - delay: "{{ state_attr(trigger.event.data.entity_id,'off_delay_seconds') | default(0) | int | timestamp_custom('%H:%M:%S',False) }}"
+      - service: system_log.write
+        data_template:
+          message: >
+            at=delay_end
+            entity_id={{ trigger.event.data.entity_id }}
+          logger: packages.lights
+          level: info
```

Comment:
Sometimes lights stay on forever, and the logs indicate that the script intended to turn them off never logs an `at=delay_end` entry. It's like the delay happens and then 💥 ?